lsp-pyright.el

```diff
@@ -152,7 +163,9 @@ set as `python3' to let ms-pyls use python 3 environments."
    ("python.analysis.typeCheckingMode" lsp-pyright-typechecking-mode)
    ("python.analysis.logLevel" lsp-pyright-log-level)
    ("python.analysis.autoSearchPaths" lsp-pyright-auto-search-paths)
-   ("python.pythonPath" lsp-pyright-locate-python)))
+   ("python.analysis.extraPaths" lsp-pyright-extra-paths)
+   ("python.pythonPath" lsp-pyright-locate-python)
+   ("python.venvPath" lsp-pyright-locate-venv)))
```

Comment:
I personally think maybe it is not a good idea to make `venvPath` configurable via emacs. Because configuring `venvPath` alone doesn't do anything, it has be used in conjunction with `venv` and `executionEnvironemnts[].venv` configurations, which can only be configured via `pyrightconfig.json` now.

If the user wants to configure `venvPath` they have to do it in `pyrightconfig.json`. 

https://github.com/microsoft/pyright/issues/702#issuecomment-636250085
https://github.com/emacs-lsp/lsp-mode/pull/1921#issuecomment-659834264

Reply:
However no objection in merging this.

Reply:
will think over it.

Reply:
You are right! I will remove venv setting.